aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -150,6 +187,167 @@ module aptos_framework::coin {
     /// Capability required to burn coins.
     struct BurnCapability<phantom CoinType> has copy, store {}
 
+    /// The mapping between coin and fungible asset.
+    struct CoinConversionMap has key {
```

Comment:
Can we add a resource group annotation here in case there are more resources later on?

Reply:
That's another layer of abstraction. Would this affect perf? I don't think we'll add more resources for this purpose.